src/app/legacy/containers/RadioSchedule/index.test.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import arabicRadioScheduleData from '#data/arabic/bbc_arabic_radio/schedule.json';
+import podcastProgramme from '#data/gahuza/bbc_gahuza_radio/p07yh8hb.json';
```

Comment:
We weren't able to find updated data for that specific article, and we wanted to use our newer response just to double check that our changes wouldn't affect the existing RadioSchedule component, since it will now effectively be using this newer format from the BFF.